sources/packages/backend/apps/api/src/route-controllers/program-year/_tests_/e2e/program-year.students.controller.getProgramYears.e2e-spec.ts

```diff
@@ -0,0 +1,157 @@
+            expect.objectContaining({
+              id: 2,
+              description:
+                "(2022-2023) - Study starting between August 01, 2022 and July 31, 2023",
+            }),
+            expect.objectContaining({
+              id: 3,
+              description:
+                "(2023-2024) - Study starting between August 01, 2023 and July 31, 2024",
+            }),
+          ]),
+        );
+      });
+  });
+
+  it("Should not include inactive program years in the list.", async () => {
+    // Arrange
+    const programYearRepo = appDataSource.getRepository(ProgramYear);
+    const programYear = await programYearRepo.findOneBy({ id: 2 });
+    programYear.active = false;
+    await programYearRepo.save(programYear);
+    const endpoint = "/students/program-year/options-list";
+    const studentToken = await getStudentToken(
+      FakeStudentUsersTypes.FakeStudentUserType1,
+    );
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(studentToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .then((response) => {
+        const programYearList = response.body;
+        expect(programYearList).toHaveLength(2);
+        expect(programYearList).toEqual(
+          expect.arrayContaining([
+            expect.objectContaining({
+              id: 1,
+              description:
+                "(2021-2022) - Study starting between August 01, 2021 and July 31, 2022",
+            }),
+            expect.objectContaining({
+              id: 3,
+              description:
+                "(2023-2024) - Study starting between August 01, 2023 and July 31, 2024",
+            }),
+          ]),
+        );
+      });
+
+    // Revert
+    programYear.active = true;
+    await programYearRepo.save(programYear);
+  });
+
+  it("Should get an active program year by id.", async () => {
+    // Arrange
+    const endpoint = "/students/program-year/1/active";
+    const studentToken = await getStudentToken(
+      FakeStudentUsersTypes.FakeStudentUserType1,
+    );
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(studentToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.OK)
+      .then((response) => {
+        const programYear = response.body;
+        expect(programYear).toEqual({
+          id: 1,
+          formName: "SFAA2021-22",
+          programYear: "2021-2022",
+          programYearDesc:
+            "Study starting between August 01, 2021 and July 31, 2022",
+        });
+      });
+  });
+
+  it("Should return 'not found' status for an inexistent program year.", async () => {
+    // Arrange
+    const endpoint = "/students/program-year/4/active";
+    const studentToken = await getStudentToken(
+      FakeStudentUsersTypes.FakeStudentUserType1,
+    );
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .get(endpoint)
+      .auth(studentToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.NOT_FOUND);
```

Comment:
Please be precise about the message, as shown below.
![image](https://user-images.githubusercontent.com/61259237/225447352-b666755f-62cc-4f55-855e-9f3a993a0fb2.png)